src/components/apphost.js

```diff
@@ -108,84 +108,118 @@ define(["appSettings", "browser", "events", "htmlMediaHelper"], function(appSett
-        }(); - 1 === supportedFeatures.indexOf("htmlvideoautoplay") && !1 !== supportsHtmlMediaAutoplay() && require(["autoPlayDetect"], function(autoPlayDetect) {
-        autoPlayDetect.supportsHtmlMediaAutoplay().then(function() {
-            appSettings.set(htmlMediaAutoplayAppStorageKey, "true"), supportedFeatures.push("htmlvideoautoplay"), supportedFeatures.push("htmlaudioautoplay")
-        }, function() {
-            appSettings.set(htmlMediaAutoplayAppStorageKey, "false")
-        })
+        isHidden || (isHidden = true, console.log("app is hidden"));
+    }
+
+    var htmlMediaAutoplayAppStorageKey = "supportshtmlmediaautoplay0";
+    var supportedFeatures = function() {
+        var features = [];
+        navigator.share && features.push("sharing");
+        browser.edgeUwp || browser.tv || browser.xboxOne || browser.ps4 || features.push("filedownload");
+        browser.operaTv || browser.tizen || browser.orsay || browser.web0s
+            ? features.push("exit")
+            : (features.push("exitmenu"), features.push("plugins"));
+        browser.operaTv || browser.tizen || browser.orsay || browser.web0s || browser.ps4 || (features.push("externallinks"), features.push("externalpremium"));
+        browser.operaTv || features.push("externallinkdisplay");
+        supportsVoiceInput() && features.push("voiceinput");
+        !browser.tv && !browser.xboxOne && browser.ps4, supportsHtmlMediaAutoplay() && (features.push("htmlaudioautoplay"), features.push("htmlvideoautoplay"));
+        browser.edgeUwp && features.push("sync");
+        supportsFullscreen() && features.push("fullscreenchange");
+        (browser.chrome || browser.edge && !browser.slow) && (browser.noAnimation || browser.edgeUwp || browser.xboxOne || features.push("imageanalysis"));
+        (browser.tv || browser.xboxOne || browser.ps4 || browser.mobile) && features.push("physicalvolumecontrol");
+        browser.tv || browser.xboxOne || browser.ps4 || features.push("remotecontrol");
+        browser.operaTv || browser.tizen || browser.orsay || browser.web0s || browser.edgeUwp || features.push("remotevideo");
+        features.push("otherapppromotions");
+        features.push("targetblank");
+        browser.orsay || browser.tizen || browser.msie || !(browser.firefox || browser.ps4 || browser.edge || cueSupported()) || features.push("subtitleappearancesettings");
+        browser.orsay || browser.tizen || features.push("subtitleburnsettings");
+        browser.tv || browser.ps4 || browser.xboxOne || features.push("fileinput");
+        browser.chrome && features.push("chromecast");
+        return features;
+    }();
+    if (supportedFeatures.indexOf("htmlvideoautoplay") === -1 && supportsHtmlMediaAutoplay() !=== false) {
```

Comment:
```suggestion
    if (supportedFeatures.indexOf("htmlvideoautoplay") === -1 && supportsHtmlMediaAutoplay() !== false) {
```
or am I missing a new Javascript comparison?..

Reply:
Nope that definitely doesn't exist, I haven't tested this change in the web interface, only Android. I will probably start testing in web as well when I can run it separately from the server, but most of my changes will be platform agnostic. This being an exception :)